### PR TITLE
More CLI query commands

### PR DESCRIPTION
**Pull request by scholtzan**
This adds a few more commands to the bqetl CLI:

```
bqetl format    -- to format SQL files
bqetl dryrun    -- to dry run specific SQL files or all SQL files in a directory
bqetl query backfill   -- to run a backfill; has some options to specify a start_date, end_date, dry_run etc
bqetl query info       -- shows information about all or specific queries, including cost and last_updated, if users have permissions to access BigQuery
```

I moved some of the logic from `script` to `bigquery_etl` which makes it easier to reuse and test. It might be good if @jklukas could take a look at these parts.

### CODE REVIEW

File: bigquery_etl/cli/__init__.py [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 
     @click.group(commands=commands)
     @click.version_option(version=__version__)
     def group():
         """CLI tools for working with bigquery-etl."""
         pass
 
+    warnings.filterwarnings(
+        "ignore", "Your application has authenticated using end user credentials"
+    )
```
Comment (scholtzan) [inline bigquery_etl/cli/__init__.py:25]: When authenticating via `gcloud auth default`, every time `google.cloud.bigquery.Client` makes a request the following warning shows up: 

```
UserWarning: Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
  warnings.warn(_CLOUD_SDK_CREDENTIALS_WARNING)
```

This mutes this warning when using the CLI

File: bigquery_etl/format_sql/format.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
+"""Format SQL."""
```
Comment (scholtzan) [inline bigquery_etl/format_sql/format.py:1]: This was moved here from `/scripts/format_sql`

File: bigquery_etl/run_query.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
+"""
```
Comment (scholtzan) [inline bigquery_etl/run_query.py:1]: This was moved here from `scripts/run_query`

Comment (jklukas) [reply bigquery_etl/run_query.py:1]: `scripts/run_query` should be changed to call this.

**Pull request merged**